CHANGES.rst

```diff
@@ -150,6 +150,9 @@ Bug fixes
 
 - ``astropy.modeling``
 
+  - Displaying Compound model expressions when printing compound model instances
+    . [#4414]
```

Comment:
This is a sentence fragment.  It should be something like
"Fixed display of compound model expressions and components when printing compound model instances"